posts/outreachy-intro/index.qmd

```diff
@@ -43,7 +43,7 @@ My name is Goodnews Sandy, I am a Frontend Developer and a Technical Writer. I e
  I have an educational background in Petroleum Engineering and had a career change to Computer Science related field because i realize that technological fields offered more opportunities to use my creativity and I love building things!!! and I do hope to use my skills to contribute to the Oil and Gas industry.
 
 ### As an Outreachy Intern
-I am one of the successful candidates selected to participate in the [Outreachy 2023 Internship](https://www.outreachy.org/) , I applied to Outreachy because I see it as an opportunity to learn new technologies and contribute to varities of community through Open Source.  The Outreachy Internship is a three months program to contribute to any Open Source community of your choice. For the selection period, I chose [Open Science Community Saudi Arabia (OSCSA)](https://github.com/Open-Science-Community-Saudi-Arabia) because their goals were clearly stated.  I contributed to improving the Open Science Community Saudi Arabia (OSCSA) [website](https://github.com/Open-Science-Community-Saudi-Arabia/OSCSA_Website) and also colloborated in developing a [Massive Open Online Courses(MOOCs)](https://github.com/Open-Science-Community-Saudi-Arabia/MOOCs/tree/dev_team2) platform. The mentors were super helful and I enjoyed contributing to the community.  
+I am one of the successful candidates selected to participate in the [Outreachy 2023 Internship](https://www.outreachy.org/) , I applied to Outreachy because I see it as an opportunity to learn new technologies and contribute to varities of community through Open Source.  The Outreachy Internship is a three months program to contribute to any Open Source community of your choice. For the selection period, I chose [Open Science Community Saudi Arabia (OSCSA)](https://github.com/Open-Science-Community-Saudi-Arabia) because their goals were clearly stated.  I contributed to improving the Open Science Community Saudi Arabia (OSCSA) [website](https://github.com/Open-Science-Community-Saudi-Arabia/OSCSA_Website) and also collaborated in developing a [Massive Open Online Courses(MOOCs)](https://github.com/Open-Science-Community-Saudi-Arabia/MOOCs/tree/dev_team2) platform. The mentors were super helful and I enjoyed contributing to the community.  
```

Comment:
```suggestion
I am one of the successful candidates selected to participate in the [Outreachy 2023 Internship](https://www.outreachy.org/) , I applied to Outreachy because I see it as an opportunity to learn new technologies and contribute to varieties of communities through Open Source.  The Outreachy Internship is a three months program to contribute to any Open Source community of your choice. For the selection period, I chose [Open Science Community Saudi Arabia (OSCSA)](https://github.com/Open-Science-Community-Saudi-Arabia) because their goals were clearly stated.  I contributed to improving the Open Science Community Saudi Arabia (OSCSA) [website](https://github.com/Open-Science-Community-Saudi-Arabia/OSCSA_Website) and also collaborated in developing a [Massive Open Online Courses(MOOCs)](https://github.com/Open-Science-Community-Saudi-Arabia/MOOCs/tree/dev_team2) platform. The mentors were super helpful and I enjoyed contributing to the community.  
```